CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -132,7 +131,7 @@ job_setup.postscript_1 = samweb file-lineage parents `basename ${fname}` > paren
 job_setup.postscript_2 = CORSIKA_EXE=`which corsika77400Linux_QGSJET_fluka`
 job_setup.postscript_3 = DATDIR=`dirname $CORSIKA_EXE`
 job_setup.postscript_4 = IFS='.' read -r -a FIELDS <<< "`cat filename.txt`"
-job_setup.postscript_5 = sed -e "s:_DATDIR_:$DATDIR/:" -e "s:_DIRECT_:`pwd`/:" -e "s:_SEED1_:$RANDOM:" -e "s:_SEED2_:$RANDOM:" -e "s:_NSHOW_:3000000:" -e "s:RUNNR   1:RUNNR   ${FIELDS[4]}:" /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/current/link/Production/JobConfig/cosmic/CORSIKAInput.txt > corsika_conf.txt
+job_setup.postscript_5 = sed -e "s:_DATDIR_:$DATDIR/:" -e "s:_DIRECT_:`pwd`/:" -e "s:_SEED1_:$RANDOM:" -e "s:_SEED2_:$RANDOM:" -e "s:_NSHOW_:3000000:" -e "s:RUNNR   1:RUNNR   ${FIELDS[4]}:" /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s/link/Production/JobConfig/cosmic/CORSIKAInput.txt > corsika_conf.txt
```

Comment:
is the 's' after %(release) intentional?

Reply:
Yes, it's the format of the ini variables

Reply:
It doesn't look like the others: its inside the directory path, not an argument.

Reply:
It doesn't matter, the system just replace all the % variables, e.g. look at the dataset defintions:
```
job_output_2.add_to_dataset = sim.%(submitter)s.CosmicDSStopsCORSIKA.%(desc)s.art
```

Reply:
OK, thanks for the explanation.